ros_ws/src/teleoperation/include/joystick_map.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <ros/ros.h>
+
+#include <sensor_msgs/Joy.h>
+
+class JoystickMap
+{
+    protected:
+    JoystickMap()
+    {
+    }
+
+    public:
+    virtual ~JoystickMap()
+    {
+    }
```

Comment:
A virtual class should always have a virtual destructor. It can be omitted, but should not. Also instead of
```C++
virtual ~JoystickMap()
{
}
```
one could write
```C++
virtual ~JoystickMap() = default;
```